client/components/app.js

```diff
@@ -25,57 +23,44 @@ class App extends Component {
   }
 
   onClickPlanTrip() {
-    this.setState({ clickPlan: !this.state.clickPlan });
+    this.setState((prevState) => ({ clickPlan: !prevState.clickPlan }));
   }
 
   onClickGetTrips() {
-    this.setState({ clickTrips: !this.state.clickTrips });
+    this.setState((prevState) => ({ clickTrips: !prevState.clickTrips }));
   }
 
-  // componentDidMount() {
-  //   // established axios connection to backend
-  //   // GET
-  //   axios
-  //     .get('/get')
-  //     .then((res) => console.log('Response data:', res))
-  //     .catch((err) => console.error(err));
-
-  //   // POST
-  //   axios
-  //     .post('/post', {
-  //       first_name: 'Josh',
-  //       last_name: 'Nunez',
-  //       email: 'joshjnunez09@gmail.com',
-  //       profile_pic:
-  //         'https://ca.slack-edge.com/T02P3HQD6-URPNXJK3R-3bab56848cef-512',
-  //       host: false,
-  //     })
-  //     .then((res) => console.log('POSTED:', res))
-  //     .catch((err) => console.error(err));
-  // }
-
   responseGoogle(response) {
     console.log('google response:', response);
```

Comment:
never use console.log, you can change your eslint to allow for warn info and error, should never have log in production code